src/Bridge.js

```diff
@@ -48,13 +48,12 @@ function useInitialRoutes() {
     ];
   }
 
-  const landingPage = hasImpliedTicket
-    ? ROUTE_NAMES.ACTIVATE
-    : ROUTE_NAMES.LOGIN;
-
-  const extraRoutes = !hasDisclaimed ? [{ key: ROUTE_NAMES.DISCLAIMER }] : [];
-
-  return [{ key: landingPage }, ...extraRoutes];
+  return hasImpliedTicket
+    ? [{ key: ROUTE_NAMES.ACTIVATE }]
+    : [
+        { key: ROUTE_NAMES.LOGIN },
+        ...(!hasDisclaimed ? [{ key: ROUTE_NAMES.DISCLAIMER }] : []),
```

Comment:
was this reverted or something? left version is much more readable

Reply:
This is doing something different though. It's only adding the disclaimer to the route for the login case, not the activate case. The previous version was adding it for both.

The js too fancy here?

Reply:
ah, didn't notice the new logic. it's fancy, but I'd say it's a-ok

-matt